.gitignore

```diff
@@ -1,7 +1,6 @@
 pkg/
 Gemfile.lock
 Gemfile.local
-vendor/
```

Comment:
Agreed.  I want to test that we don't create any temp files in there and may get checked in by accident.